code/modules/clothing/glasses/_glasses.dm

```diff
@@ -307,7 +307,15 @@
 	name = "beer goggles"
 	icon_state = "sunhudbeer"
 	desc = "A pair of sunglasses outfitted with apparatus to scan reagents, as well as providing an innate understanding of liquid viscosity while in motion."
```

Comment:
akhem

Reply:
I was meaning to get to this but I forgot.  Fixed with 4101513c2f08147570369c89352774779cc7e556 ty